### ISSUE TITLE
Issue rendering links

### ISSUE THREAD

**Issue by rupertrutland** [probablyup/markdown-to-jsx#163]
Try passing through the following markdown:

```
[http://foo](http://bar)
```

The eventual markup result is invalid HTML (nested anchor tags):
```
<a href="http://bar" class="">
  <a href="http://foo" class="">http://foo</a>
</a>
```

This is taken from the demo website https://probablyup.github.io/markdown-to-jsx/

![image](https://user-images.githubusercontent.com/1215981/37422929-ce1637a2-27bc-11e8-889b-feee50741bc2.png)

**Comment by probablyup** [comment]
Good catch, will take a look in the next day or two

**Comment by probablyup** [comment]
Released as `6.5.2`, thanks for reporting!
